access/resource_permissions_test.go

```
@@ -419,3 +422,76 @@ func TestResourcePermissionsCreate_error(t *testing.T) {
 		}
 	}
 }
+
+func TestAccAddOrModifyDeleteJobPermissions(t *testing.T) {
+	cloudEnv := os.Getenv("CLOUD_ENV")
+	randomName := qa.RandomName()
+
+	if cloudEnv == "" {
+		t.Skip("Acceptance tests skipped unless env 'CLOUD_ENV' is set")
+	}
+
+	client := common.CommonEnvironmentClient()
+
+	jobSettings := compute.JobSettings{
+		NewCluster: &compute.Cluster{
+			NumWorkers:   2,
+			SparkVersion: "6.4.x-scala2.11",
+			SparkConf:    nil,
+			NodeTypeID:   "Standard_DS3_v2",
+		},
+		Name: "Job " + randomName,
+	}
+	jobCreate, _ := compute.NewJobsAPI(client).Create(jobSettings)
+
+	job, _ := compute.NewJobsAPI(client).Read(fmt.Sprint(jobCreate.JobID))
```

do not ignore errors (`, _`) and add `assert.NoError(t, err)` everywhere.